back/app/policies/user_policy.rb

```diff
@@ -103,16 +103,10 @@ def view_private_attributes?
     !!((user && (instance&.id == user.id || user.admin?)) || instance&.invite_pending?)
   end
 
-  def permitted_attributes_for_create
+  def permitted_attributes
```

Comment:
I think we could just rename `shared_permitted_attributes` to `permitted_attributes` and add the email there?

Reply:
Unfortunately, this causes a FranceConnect test to fail, so i think I'll go with my previous version:

```
1) Users when authenticated when non-admin PUT web_api/v1/users/:id  Can't change some attributes of a user verified with FranceConnect
     Failure/Error: expect(@user.first_name).not_to eq first_name
     
       expected: value != "Raymond"
            got: "Raymond"
     
       (compared using ==)
     # ./spec/acceptance/users_spec.rb:1114:in `block (6 levels) in <top (required)>'
     # /bundle/ruby/2.7.0/gems/webmock-3.18.1/lib/webmock/rspec.rb:37:in `block (2 levels) in <main>'
  ```